commons-business-impl/pom.xml

```diff
@@ -43,6 +43,12 @@
 			<artifactId>commons-model</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+		<dependency>
+		    <groupId>org.projectlombok</groupId>
+		    <artifactId>lombok</artifactId>
+		    <version>1.18.12</version>
```

Comment:
Shouldn't we manage lombok version in dependency management section of parent pom.xml?

Reply:
Yes. https://github.com/AppJars/commons-backend/issues/28